lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+
+  attr_reader :meals
+
+  def initialize
+    @meals = {
```

Comment:
I think this may be easier as a Hash of Arrays, with Strings as key values

Reply:
Yes I think so too, I will make the changes later on.